docs/src/man/hosting.md

```diff
@@ -535,6 +537,90 @@ Preview builds are still deployed to the `previews` subfolder.
     ([source repository](https://github.com/JuliaDocs/juliadocs.github.io)) is one example
     where this functionality is used.
 
+
+## Deploying from a monorepo
+
+Documenter.jl supports building documentation for a package that lives in a monorepo, e.g., in a repository that contains multiple packages (including one potentially top level-)
+
+Here's one example of setting up documentation for a repository that has the following structure: one top level package and two subpackages PackageA.jl and PackageB.jl:
+```
+.
+├── README.md
+├── docs
+|   ├── make.jl
+│   └── Project.toml
+├── src/...
+├── PackageA.jl
+│   ├── docs
+|   │   ├── make.jl
+|   │   └── Project.toml
+│   └── src/...
+└── PackageB.jl
+    ├── docs
+    │   ├── make.jl
+    │   └── Project.toml
+    └── src/...
+```
+
+The three respective `make.jl` scripts should contain [`deploydocs`](@ref) settings that look something like
+
+```
+# In ./docs/make.jl
+deploydocs(; repo = "github.com/USER_NAME/PACKAGE_NAME.jl.git",
+            # ...any additional kwargs
+            )
+
+# In ./PackageA.jl/docs/make.jl
+deploydocs(; repo = "github.com/USER_NAME/PACKAGE_NAME.jl.git",
+             dirname="PackageA",
+             tag_prefix="PackageA-",
```

Comment:
I haven't followed TagBot lately. Does TagBot generate tags like this by default, or is it configurable on that end too?

Reply:
I don't think TagBot supports subdir packages at all (xref https://github.com/JuliaRegistries/TagBot/issues/157), but this PR would allow manual tags at least. Ideally tagbot could be updated too, although @christopher-dG labelled that issue with "rewrite" so maybe it would need a lot of work.

Reply:
Correct, tbh if that feature is added (whether in a full rewrite or not) it is unlikely that I will be the one who does it.

Reply:
Opened a PR to add this feature: https://github.com/JuliaRegistries/TagBot/pull/252/files

(I thought it would be a fun way to learn how GHA are written, and instead fought to remember python. you win some, you lose some!)